src/ethereum_test_tools/tests/test_filling/test_expect.py

```diff
@@ -0,0 +1,234 @@
+"""
+Test suite for `ethereum_test_tools.filling` fixture post (expect) section.
+"""
+from typing import Any, Mapping, Optional
+
+import pytest
+
+from ethereum_test_forks import Shanghai
+from evm_transition_tool import FixtureFormats, GethTransitionTool
+
+from ...common import Account, Environment, TestAddress, Transaction, to_address
+from ...common.types import Storage
+from ...spec import StateTest
+
+# from ...spec.blockchain.types import FixtureCommon as BlockchainFixtureCommon
+
+ADDRESS_UNDER_TEST = to_address(0x01)
+pre = {
+    TestAddress: Account(balance=1000000000000000000000),
+}
+post = {}
+
+
+def run_post_state_mismatch_test(
+    pre: Mapping[Any, Any], post: Mapping[Any, Any], exception: Any
+) -> Optional[pytest.ExceptionInfo]:
+    """
+    Performs test filling and post state verification.
+    """
+    fork = Shanghai
```

Comment:
Is there a good alternative to hard-coding the fork here? :thinking: 

How about using `ethereum.test_forks.get_deployed_forks()[-1]`?